backend/app/transcendence/src/main.ts

```diff
@@ -0,0 +1,15 @@
+import { NestFactory } from "@nestjs/core";
+import { AppModule } from "./app.module";
+
+async function bootstrap()
+{
+	const app = await NestFactory.create(AppModule);
+	await app.listen(3000);
+}
+
+bootstrap();
+
+/**
+ * Usefull documentation I've found here 
+ * https://www.thisdot.co/blog/introduction-to-restful-apis-with-nestjs/
+ */

```

Comment:
This is useful but probably not in the proper place, this belongs to the wiki with some explanations as to why and how it's useful. What does it help us with, what did you understand because of it that you didn't, why do you think we should read it, what do you think it would teach us etc.